dms/tests/test_benchmark.py

```diff
@@ -2,25 +2,81 @@
 # Copyright 2020 Creu Blanca
 # License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
 
+import functools
 import logging
+import threading
+import time
 import unittest
 
 from odoo.modules.module import get_module_resource
 from odoo.tests import common, tagged
 from odoo.tools import convert_file
 from odoo.tools.misc import profile
 
-from .common import track_function
-
 _logger = logging.getLogger(__name__)
 
+# ----------------------------------------------------------
+# Decorators
+# ----------------------------------------------------------
+
+
+def track_function(
```

Comment:
Why moving this function?

Reply:
Because it is only used in this test.

Reply:
Well, in this case, for reducing the diff, but also if in any moment it's needed by other test, let's keep it where it was. It's not an overhead having it there. It's loaded once the same being in one specific test or in the common one.